tests/Fixtures/ci-integration/.gitignore

```diff
@@ -0,0 +1 @@
+!/composer.lock
```

Comment:
What about using `!/**`? If I'm not wrong, this should include all files, so we won't have the same issue with other files.

Reply:
I tend to develop towards as `close` as possible and `open` when needed. In this case we don't need to open up the `ignore` fully so I would not do it. However, I'm open for other opinions.

Reply:
actually, as `close` as possible here would mean `as independent` as possible, as it's integration test.
for that, we want to be affected by global config as less as possible, isn't it right?

Reply:
It is a two way as most of the time, in my POV;
- If I open this up to `all/wildcards` and in the future something is added to a test or files or directories within in this directory that should be ignored; than this needs to be adjusted. 
- If above does not happen, than the current state of the PR is as good as the suggested open one.

However as I just want the tests to pass locally; I do not feel strongly about the solution taken as both will do.

Reply:
that's exactly the thing. we want to have git sandbox. for that, we shall not inherit from project/global git config

Reply:
updated